.github/workflows/build-windows.yml

```diff
@@ -141,16 +141,19 @@ jobs:
           # Run GTSAM_UNSTABLE tests
           cmake --build build -j4 --config ${{ matrix.build_type }} --target check.base_unstable
           # Compile. Fail with exception
-          # cmake --build build -j4 --config ${{ matrix.build_type }} --target check.geometry_unstable
+          # cmake --build build -j1 --config ${{ matrix.build_type }} --target check.geometry_unstable
```

Comment:
Any reason you changed this to j1 instead of j4? We would like to the CI to run as fast as possible.

Reply:
Yes, some of the tests are not compiling with -j4 due memory allocation or just hang. Some times we get a weak vm. (not all the vm are equal).

Reply:
Interesting. Could we instead use 2 instead of 1? That way we have some parallel execution.

Reply:
I will convert it back to -j4 because it commented